perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -283,6 +306,58 @@ def _MakeSwiftCommandPrefix(auth_url, tenant_name, username, password):
   return ' '.join(options)
 
 
+def _ComputeMultiStreamStartTimes():
+  """Compute write and read start times for the api_multistream
+benchmark.
+
+  These are the timestamps that we pass to the worker processes to
+  tell them when to go.
+
+  Returns: a tuple (write_start_time, read_start_time).
+  """
+
+  write_start_time = time.time() + MULTISTREAM_BUFFER_TIME
+  kb_to_upload = FLAGS.object_size_kb * FLAGS.objects_per_stream
+  read_start_time = (write_start_time +
```

Comment:
Let's see if it works better to express read_start_time as relative time, i.e., x seconds after all writes are complete.

When the test script on a VM gets invoked by us, it gets two time values: one is the write_start_time, which is expressed in absolute time, and then it does all the write tests using multiple streams, then it waits for all the writes to finish. After that, it checks if there is a "read_delay_time", if no, start reads immediately, if yes, wait for a read_delay_time and then start reads. This way, we would never have to guess how fast a provider can do upload to properly set the read_start_time as absolute timestamp.

To select a read_delay_time, it should be more than (perhaps 2x-3x more than) the maximum time it takes to launch the test script one by one on N VMs which is "relatively" less variable (because that would be mostly comprised of time to call RemoteCommand())